scene/theme/default_theme.cpp

```diff
@@ -773,8 +773,11 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_stylebox("custom_button_hover", "Tree", button_hover);
 
 	theme->set_icon("checked", "Tree", icons["checked"]);
+	theme->set_icon("checked_disabled", "Tree", icons["checked"]);
```

Comment:
`checked_disabled`, `unchecked_disabled` and `indeterminate_disabled` icons should be added to `scene/theme/icons` based on the existing designs in that folder.

Reply:
Hmmm, I used the same logic as for CheckBox. Do I need to fix it for CheckBox as well?
![image](https://github.com/godotengine/godot/assets/1756388/42eb10bc-c759-4a98-91b9-94a34efd8b7e)

```
theme->set_icon("checked", "CheckBox", icons["checked"]);
theme->set_icon("checked_disabled", "CheckBox", icons["checked"]);
theme->set_icon("unchecked", "CheckBox", icons["unchecked"]);
theme->set_icon("unchecked_disabled", "CheckBox", icons["unchecked"]);
```

Reply:
Nevermind, I thought the icon wasn't automatically modulated.

Reply:
The icon wasn't automatically modulated. I fixed this right now :) 
```
theme->set_icon("checked", "Tree", icons["checked"]);
theme->set_icon("checked_disabled", "Tree", icons["checked_disabled"]);
theme->set_icon("unchecked", "Tree", icons["unchecked"]);
theme->set_icon("unchecked_disabled", "Tree", icons["unchecked_disabled"]);
theme->set_icon("indeterminate", "Tree", icons["indeterminate"]);
theme->set_icon("indeterminate_disabled", "Tree", icons["indeterminate_disabled"]);
```

There are icons for disabled state `checked_disabled` and `unchecked_disabled`, but they are not used